### PR TITLE
srcpkgs/: set update to ignore development series and other incorrect versions

### PR DESCRIPTION
**Pull request by TinfoilSubmarine**
Cleans up some more update checks. I only reviewed the orphaned package list.

### CODE REVIEW

**Comment by ericonr** [comment]
Please word the commit to be `srcpkgs/: ...`

File: srcpkgs/botan/update [hunk 1]
```diff
@@ -1 +1,2 @@
 pattern='\K[0-9]+\.[0-9]*[02468]\.[0-9]+'
+ignore="*3.0.*" # alpha release series
```
Comment (ericonr) [inline srcpkgs/botan/update:2]: Will the non alpha release be 3.1.x?

Comment (TinfoilSubmarine) [reply srcpkgs/botan/update]: I just ran the update check on botan again with the `update` file deleted and it seems like the pattern was broken. It is working properly now with no need to have the ignore.

**Comment by paper42** [comment]
could you rebase and split to multiple commits?

**Pull request merged**